crates/common/src/lib.rs

```diff
@@ -24,3 +24,11 @@ cfg_if::cfg_if! {
         use alloc::vec;
     }
 }
+
+lazy_static::lazy_static! {
```

Comment:
The `gw-common` is supposed used by both godwoken node & scripts. It should only depends on `no-std` crates.

We should create a new crate `gw-ckb-hardfork` and move these code to there.